.circleci/config.yml

```diff
@@ -1,29 +1,28 @@
 version: 2.1
 
 executors:
-  zenith-build-executor:
+  zenith-xlarge-executor:
     resource_class: xlarge
     docker:
       # NB: when changed, do not forget to update rust image tag in all Dockerfiles
-      - image: cimg/rust:1.56.1
-  zenith-python-executor:
+      - image: zimg/rust:1.56
```

Comment:
I wonder is it important which version of python we use? Probably not so much, especially with poetry

Reply:
I have used 3.7.12 as @funbringer suggested, but we can setup another at corresponding [Dockerfiles](https://github.com/zenithdb/docker-images/blob/main/rust/1.56.1/Dockerfile#L56)